charts/node-analyzer/templates/_helpers.tpl

```diff
@@ -0,0 +1,209 @@
+{{- end -}}
+
+{{/*
+Return the proper Sysdig nodeAnalyzer image name for the Eve Connector
+*/}}
+{{- define "nodeAnalyzer.image.eveConnector" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.repository -}} {{- if .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.digest -}} @ {{- .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper image name for the Image Analyzer
+*/}}
+{{- define "nodeAnalyzer.image.imageAnalyzer" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.imageAnalyzer.image.repository -}} {{- if .Values.nodeAnalyzer.imageAnalyzer.image.digest -}} @ {{- .Values.nodeAnalyzer.imageAnalyzer.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.imageAnalyzer.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper image name for the Host Analyzer
+*/}}
+{{- define "nodeAnalyzer.image.hostAnalyzer" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.hostAnalyzer.image.repository -}} {{- if .Values.nodeAnalyzer.hostAnalyzer.image.digest -}} @ {{- .Values.nodeAnalyzer.hostAnalyzer.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.hostAnalyzer.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper image name for the Benchmark Runner
+*/}}
+{{- define "nodeAnalyzer.image.benchmarkRunner" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.benchmarkRunner.image.repository -}} {{- if .Values.nodeAnalyzer.benchmarkRunner.image.digest -}} @ {{- .Values.nodeAnalyzer.benchmarkRunner.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.benchmarkRunner.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Daemonset labels
+*/}}
+{{- define "daemonset.labels" -}}
+  {{- if .Values.daemonset.labels }}
+    {{- $tp := typeOf .Values.daemonset.labels }}
+    {{- if eq $tp "string" }}
+      {{- tpl .Values.daemonset.labels . }}
+    {{- else }}
+      {{- toYaml .Values.daemonset.labels }}
+    {{- end }}
+  {{- end }}
+{{- end -}}
+
+{{/*
+Node Analyzer labels
+*/}}
+{{- define "nodeAnalyzer.labels" -}}
+helm.sh/chart: {{ include "nodeAnalyzer.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+  {{- if .Values.nodeAnalyzer.labels }}
+    {{- $tp := typeOf .Values.nodeAnalyzer.labels }}
+    {{- if eq $tp "string" }}
+      {{- tpl .Values.nodeAnalyzer.labels . }}
+    {{- else }}
+      {{- toYaml .Values.nodeAnalyzer.labels }}
+    {{- end }}
+  {{- end }}
+{{- end -}}
+
+{{/*
```

Comment:
Didn't seem like this was used anywhere in these files, so if that's the case this can probably be removed as well

Reply:
this is now being used by serviceaccount-node-analyzer.yaml file